circle.yml

```diff
@@ -18,8 +18,8 @@ general:
       - 'gh-pages'
 
 deployment:
-  website:
-    branch: master
+  pluginDoc:
+    tag: /v[0-9]+(\.[0-9]+)(\.[0-9]+){3}/
```

Comment:
The regexp is wrong, as it matches v1.0.0.0.0.0.0 It should be `/v[0-9]+(\.[0-9]+){2}/`

Actually this works fine for us when building our images. The trigger here simply says, that when GitHub sends a POST because of a tag (must be of course enabled in the webhook, I will check that), then this build is triggered. `branch: master` means on every commit on master.

So this should cover our use case here.